website/components/pluginData.tsx

```diff
@@ -86,7 +86,7 @@ export const ALL_PLUGINS: Plugin[] = [
     logoDark: "/images/logos/plugins/firestore-dark.svg",
     id: "firestore",
     kind: "source",
-    availability: "free",
+    availability: "premium",
```

Comment:
There's more cleanup to be done to remove any plugin data from the old Website, but didn't want to dig too deep into it